test/jest/victory-core/victory-primitives/curve.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Curve } from "victory-line";
+import { renderInSvg } from "../../rendering-utils";
+import * as d3Scale from "victory-vendor/lib/d3-scale";
```

Comment:
This should be `victory-vendor/d3-scale` unless we're having a Jest problem with it using the ESM version like @scottrippey was diagnosing today? (Technically in Node.js you shouldn't be able to specify `victory-vendor/lib/*`, but Jest doesn't use real Node.js require's)